.travis.yml

```diff
@@ -24,3 +24,5 @@ after_success:
   - bin/createcoverage
   - pip install coverage==3.7.1 coveralls
   - coveralls
+  - docker.io build --tag plone.restapi:$(git rev-parse HEAD) .
+#  - docker.io push kitconcept/plone.restapi
```

Comment:
Oh and make sure to check: https://docs.travis-ci.com/user/docker/

You need:

```
services:
- docker
```

and change

```
docker.io
```

to ```docker```